compiler/qsc_project/src/project.rs

```diff
@@ -321,41 +380,37 @@ pub trait FileSystemAsync {
 
     /// Load the sources for a single package at the given directory. Also load its
     /// dependency information but don't recurse into dependencies yet.
+    ///
+    /// Any errors that didn't block project load are accumulated into the `errors` vector.
     async fn read_local_manifest_and_sources(
         &self,
         directory: &Path,
+        errors: &mut Vec<Error>,
     ) -> ProjectResult<PackageInfo> {
         let manifest = self.parse_manifest_in_dir(directory).await?;
 
-        let manifest = ManifestDescriptor {
-            manifest_dir: directory.to_path_buf(),
-            manifest,
-        };
+        // For local packages, we include all *.qs files under the `src/`
```

Comment:
Is there a way to intentionally omit a local .qs file from being part of a project while leaving it under `src/`?

Reply:
With this change, there won't be.